test/builder/eventlistener.go

```diff
@@ -103,6 +106,38 @@ func EventListenerReplicas(replicas int32) EventListenerSpecOp {
 	}
 }
 
+// EventListenerPodTemplate sets the specified pod template of the EventListener.
+func EventListenerPodTemplate(podTemplate v1alpha1.PodTemplate) EventListenerSpecOp {
+	return func(spec *v1alpha1.EventListenerSpec) {
+		spec.DeprecatedPodTemplate = podTemplate
+	}
```

Comment:
are there any tests using this? If not, do we even need these given that the fields are deprecated?